src/routes.js

```diff
@@ -4,6 +4,8 @@ import SignIn from './views/SignIn';
 import Yahtzee from './components/Yahtzee';
 import Nertz from './views/games/Nertz';
 import YuGiOh from './views/YuGiOh';
+import Qwixx from './components/Qwixx';
```

Comment:
We should move your Qwixx component to the /views/games folder instead of /components

Reply:
Yeah I tried to do that yesterday but then nothing would load on my Qwixx page when i tested it so I put it back in components so I could see what I was editing.

Reply:
You should just be able to move it and then update the path on this line to match.

Reply:
hmm yeah i tried that exact path and it wasnt working but i'll try it again